src/rtctools_heat_network/heat_mixin.py

```diff
@@ -2749,8 +2749,8 @@ def __ates_heat_to_discharge_path_constraints(self, ensemble_member):
         bounds = self.bounds()
 
         for a, (
-            (_cold_pipe, _cold_pipe_orientation),
             (hot_pipe, _hot_pipe_orientation),
+            (_cold_pipe, _cold_pipe_orientation),
         ) in self.heat_network_topology.ates.items():
             heat_nominal = parameters[f"{a}.Heat_nominal"]
             q_nominal = self.variable_nominal(f"{a}.Q")
```

Comment:
few line below HeatIn.Q should be HeatOut.Q or general Q added for storage components